nuxeo-runtime/nuxeo-launcher-commons/src/main/java/org/nuxeo/launcher/config/ConfigurationGenerator.java

```diff
@@ -2000,12 +2010,16 @@ public File getTemplateConf(String template) throws ConfigurationException {
 
     /**
      * Gets the Java options with 'nuxeo.*' properties substituted. It enables usage of property like ${nuxeo.log.dir}
-     * inside JAVA_OPTS.
+     * inside JAVA_OPTS. The default values are defined by {@link #JAVA_OPTS_DEFAULT}.
      *
-     * @return the java options string.
-     */
-    protected String getJavaOpts(String key, String value) {
-        return StrSubstitutor.replace(System.getProperty(key, value), getUserConfig());
+     * @return the Java options string.
+     * @see #JAVA_OPTS_DEFAULT
+     */
+    protected String getJavaOpts() {
+        String options = System.getProperty(JAVA_OPTS_PROP, JAVA_OPTS_DEFAULT);
```

Comment:
I disagree with JAVA_OPTS_DEFAULT: it is set for use in `ServerConfigurator.getInfo(String, List<LocalPackage>)` which must not guess or set any default value, it must report the effective values